StudentsDB/src/main/java/com/musala/simple/students/db/mappers/SQLObjectMapper.java

```diff
@@ -0,0 +1,31 @@
+package com.musala.simple.students.db.mappers;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.musala.simple.students.db.Student;
+
+public class SQLObjectMapper {
```

Comment:
You removed the base ObjectMapper interface, but it would be good to bring it back, so that it defines the `toStudentObj` method :) The reason is that in the future we may have some other kind of objects, s.g. School. So, if I was to implement this, I would first add the new method `toSchoolObj` in the interface and the IDE will show me where I have to provide implementation (in Mongo and MySQL wrapper). So, this will bring more readability and abstraction.